spec.bs

```diff
@@ -56,18 +55,17 @@ spec: web-authn; urlPrefix: https://w3c.github.io/webauthn/
 
 This specification defines an API that enables the use of strong authentication
 methods in payment flows on the web. It aims to provide the same authentication
-benefits and user privacy focus as [[webauthn-3]], whilst relaxing certain
-constraints to meet the needs of payment processing.
-
-Similarly to [[webauthn-3]], this specification defines two related processes
-involving a user. The first is [[#sctn-enrollment]], where a relationship is
-created between the user and the [=Relying Party=]. The second is
-[[#sctn-authentication]], where a specific payment from the user to a
-recipient (possibly via an intermediary payment service provider) is
-authenticated for the [=Relying Party=]. An important concept in Secure Payment
-Confirmation is that with the permission of the [=Relying Party=], the
-merchant or another entity may initiate an authentication ceremony on the
-[=Relying Party's=] behalf.
+benefits and user privacy focus as [[webauthn-3]] with enhancements to meet the needs of payment processing.
+
+Similarly to [[webauthn-3]], this specification defines two related
+processes involving a user. The first is [[#sctn-enrollment]], where a
+relationship is created between the user and the [=Relying
+Party=]. The second is [[#sctn-authentication]], where the user
+responds to a challenge from the [=Relying Party=] (possibly via an
+intermediary payment service provider) to consent to a specific
+payment. An important feature of Secure Payment Confirmation is that
+the merchant (or another entity) may initiate the authentication
+ceremony on the [=Relying Party's=] behalf and with their permission.
```

Comment:
Alas, 'with their permission' is not true. Nothing stops merchants from sharing 3p credentials, at which point anybody may initiate the ceremony. It's not **useful** for the entity that is initiating it, but they technically can.

Reply:
The current text is: "An important concept in Secure Payment Confirmation is that with the permission of the Relying Party...."

I preserved "with their permission" but have no problem dropping it.

Having said that, "permission" is still relevant even if it might not be the ideal term. The RP would have to share the credential ID, which is a form of permission. 

Are you ok to simply drop "and with their permission"?

Reply:
Ahaha, touche. I apologise, and send the original criticism to myself instead.

We can drop 'and with their permission'